1.0/docs/devguide/styling.md

```diff
@@ -557,9 +572,63 @@ Polymer elements (for example, in `<style>` elements within a custom element's
 the document level. **The `custom-style` extension should only be used for
 defining document styles, outside of a custom element's local DOM.**
 
+## Shared styles and external stylesheets {#style-modules}
+
+To share style declarations between elements, create a _style module_.
+A style module declares a named set of style rules that can be imported into 
+an element definition, or into a `custom-style` element.
+
+**Note:** Style modules were introduced in {{site.project_title}} 1.1;
+they replace the experimental support for [external stylesheets](#external-styleshees).
+{: .alert .alert-info }
+
+Define a style module inside an HTML import using the `<dom-module>` 
+element. 
+
+    <!-- shared-styles.html -->
+    <dom-module id="shared-styles">
+      <template>
+        <style>
+          .red { color: red; }
+        </style> 
+      </template>
+    </dom-module>
+
+The `id` attribute specifies the name you'll use to reference
+the style module. Style module names use the same namespace as elements, 
+so your style modules must have unique names.
+
+Using the shared style module is a two-step process: you need to use a `<link>` tag
+to _import_ the module, and a `<style>` tag to _include_ the styles in the correct place.
+
+To use a style module in an element:
+
+    <!-- import the module  -->
+    <link rel="import" href="../shared-styles/shared-styles.html">
+    <dom-module id="x-foo">
+      <template>
+        <!-- include the style module by name -->
+        <style include="shared-styles"></style>
+        <style>:host { display: block; }</style>
```

Comment:
Do the included styles go before or after the styles declared in the body of the element?

I'm guessing before...? Anyone know?